config/mattermost-push-proxy.sample.json

```diff
@@ -38,7 +38,6 @@
             "ServiceFileLocation":""
         }
     ],
-    "EnableConsoleLog": true,
-    "EnableFileLog": false,
-    "LogFileLocation": ""
+    "LoggingCfgFile": "",
+    "LoggingCfgJSON": ""
```

Comment:
Please see my other comment. We can keep these existing config settings, and just add a new one.